Makefile

```diff
@@ -22,6 +22,7 @@ parallel_tests:=$(filter-out $(serial_tests),$(tests))
 test: $(tests)
 	coverage combine
 	rm -f .coverage.*
+	$(MAKE) -C daemons import-test
```

Comment:
The `test` target is not invoked by PR or branch builds Travis. Only the scheduled IT builds invoke it. Maybe you could make this a separate target and have `test` and `parallel_test` depend on it?

Reply:
I can do that, but I'd like us to fix our test system. I think what we have now is utterly confusing.

Let's make `make test` run all tests, serial then parallel, and always invoke it to run any tests. To make parallel travis stages work, we can set environment variables listing the test files to include or exclude, and put logic in the makefile to default the former to `tests/test_*.py` and use filter-out to process the latter.

Reply:
I implemented your suggestion, so parallel_test should now run this too.